packages/shared/src/components/post/ShareYouTubeContent.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { ReactElement } from 'react';
+import PostSourceInfo from './PostSourceInfo';
+import { Post } from '../../graphql/posts';
+import { SharePostTitle } from './share';
+import { SharedLinkContainer } from './common/SharedLinkContainer';
+import { SharedPostLink } from './common/SharedPostLink';
+import { YouTubeVideo } from './common/YouTubeVideo';
+
+interface ShareYouTubeContentProps {
+  post: Post;
+  onReadArticle: () => Promise<void>;
+}
+
+function ShareYouTubeContent({ post }: ShareYouTubeContentProps): ReactElement {
```

Comment:
This is for the shared post card?
I'm a bit confused by the naming so not 100% sure here.

Reply:
This is still for the post modal for shared youtube video. You will notice there are two Sketch boards for Post Modal that is because there can be two sources. Either directly from YT or a link shared through a Squad.